### ISSUE TITLE
Increased precision in Activity log?

### ISSUE THREAD

**Issue by Boulder08** [RainbowMiner/RainbowMiner#2748]
Would it be possible to increase the precision of the displayed value for mBTC/kWh in the activity log? It would make it easier to determine if the switching parameters are too loose. Also the "Earned" figure is mostly zero with GPU mining on something like a 3070, so it would be nice to see more details there as well.

![image](https://github.com/RainbowMiner/RainbowMiner/assets/11799308/12851676-01ec-4088-a9ae-2c9322460e2b)

**Comment by RainbowMiner** [comment]
Yes, that would be possible. I could adjust the value to sat/kWh

**Comment by RainbowMiner** [comment]
Implemented - the activity log value was adjusted for better precision. Closing as completed - thanks for the report!
